modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryImpl.java

```diff
@@ -303,7 +303,7 @@ public int boundPort() throws IgniteSpiException {
     /**
      * @return connection check interval.
      */
-    public long connectionCheckInterval() {
+    long connectionCheckInterval() {
```

Comment:
why changed?

Reply:
To increase encapsulation once we touched it. Internal usages inly, not an interface.

Reply:
We shouldn't reduce visibility of methods in public API. It is highly unlikely of course that there are users calling this method directly but this concealing of method is definitely a breaking change, it could be targeted only to major releases not a minor ones.

Reply:
> We shouldn't reduce visibility of methods in public API. It is highly unlikely of course that there are users calling this method directly but concealing of method is definitely a breaking change. We could target it to 3.0 version but in minor releases it is not acceptable.

Fixed.